app/forms/curation_concerns/forms/work_form.rb

```diff
@@ -44,8 +44,6 @@ class << self
         # 'rights' which only has a single value on the form.
         def multiple?(term)
```

Comment:
Now that this method is only special-casing `'ordered_member_ids'`, would you consider refactoring this method to use the guard clause pattern?

```ruby
def multiple?(term)
  return true if term.to_s == 'ordered_member_ids'
  super
end
```